src/clientapi/bootstrap.js

```diff
@@ -0,0 +1,89 @@
+/** @license
+ * This file is part of the Game Closure SDK.
+ *
+ * The Game Closure SDK is free software: you can redistribute it and/or modify
+ * it under the terms of the Mozilla Public License v. 2.0 as published by Mozilla.
+
+ * The Game Closure SDK is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Mozilla Public License v. 2.0 for more details.
+
+ * You should have received a copy of the Mozilla Public License v. 2.0
+ * along with the Game Closure SDK.  If not, see <http://mozilla.org/MPL/2.0/>.
+ */
+import {
+  GLOBAL,
+  merge
+} from 'base';
+
+import analytics from 'analytics';
+// TODO: introduce some logic to import desired social platform
```

Comment:
@bchevalier it looks like we’re including fbinstant baked into devkit-core now? We will very soon need to act on this TODO.

Reply:
fbinstant is added as a script dependency in the `index.html` file generated by the build